packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterValue/DimensionFilterAutocomplete.tsx

```diff
@@ -64,28 +20,35 @@ export const DimensionFilterAutocomplete = (
     multiple,
     name,
     fieldOnChange,
-    values,
     disabled,
     fieldOnBlur,
-    isError,
-    isLoading,
     placeholderText,
     errorText,
     fieldValue,
+    serviceType,
+    dimensionLabel,
+    values,
   } = props;
 
+  const options = useMemo(
+    () =>
+      getStaticOptions(
+        serviceType ?? undefined,
+        dimensionLabel ?? '',
+        values ?? []
```

Comment:
```suggestion
        values ?? []
```
since you are sending as empty fallback value so [this](https://github.com/ACLPManager/manager/blob/44ac5ec199296e2fd1c4c9abb0a953f4a3de639e/packages/manager/src/features/CloudPulse/Alerts/CreateAlert/Criteria/DimensionFilterValue/utils.ts#L83) will never be null, so its better we update it to avoid returning fallbacks